RELEASE.md

```diff
@@ -11,22 +11,22 @@ repository](https://github.com/jupyterhub/dockerspawner).
 
 ## Steps to make a release
 
-1. Checkout master and make sure it is up to date.
+1. Checkout main and make sure it is up to date.
 
    ```bash
    ORIGIN=${ORIGIN:-origin} # set to the canonical remote, e.g. 'upstream' if 'origin' is not the official repo
-   git checkout master
-   git fetch $ORIGIN master
-   git reset --hard $ORIGIN/master
+   git checkout main
+   git fetch $ORIGIN main
+   git reset --hard $ORIGIN/main
    # WARNING! This next command deletes any untracked files in the repo
    git clean -xfd
    ```
 
-1. Update [CHANGELOG.md](CHANGELOG.md) and add it to
+1. Update [docs/source/changelog.md](docs/source/changelog.md) and add it to
    the working tree.
 
    ```bash
-   git add CHANGELOG.md
+   git add docs/source/changelog.md
    ```
 
    Tip: Identifying the changes can be made easier with the help of the
```

Comment:
Since `github-activity` is now part of the `executablebooks` project (https://github.com/executablebooks/github-activity), maybe we can update the link text and URL? It's sightly unrelated to the PR, and it redirects to the right GitHub repo as it is, but I figured it would make the `RELEASE` file complete.

Reply:
Agree! I pushed a commit about that!